frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketForm.component.tsx

```diff
@@ -25,16 +26,19 @@ import { CommentsPanel } from './commentsPanel/commentsPanel.component';
 const SCROLLBAR_ID = 'cardScrollbar';
 interface ModulePanelProps {
 	module: TemplateModule;
-	moduleValues: Record<string, any>;
-	scrollPanelIntoView: (isExpanding, el) => void;
+	scrollPanelIntoView: (isExpanding, event) => void;
 	defaultExpanded: boolean;
 }
 
-const ModulePanel = ({ module, moduleValues, scrollPanelIntoView, defaultExpanded, ...rest }: ModulePanelProps) => (
-	<Accordion {...getModulePanelTitle(module)} onChange={scrollPanelIntoView} defaultExpanded={defaultExpanded}>
-		<PropertiesList module={`modules.${module.name || module.type}`} properties={module.properties || []} propertiesValues={moduleValues} {...rest} />
-	</Accordion>
-);
+const ModulePanel = ({ module, scrollPanelIntoView, defaultExpanded, ...rest }: ModulePanelProps) => {
+	const required = some(module.properties, (property) => property.required);
```

Comment:
`lodash` methods offer a shortcut where you just pass the name of the property you want to do stuff about instead of a function. In this case, for instance, you can write `some(module.properties, 'required')` and it should work. However, if you want to pass the function `(property) => property.required)` I would go for the built-in Array.prototype.some as its more intuitive

Reply:
ahh, that's good to know